controllers/comments.controller.js

```diff
@@ -44,3 +46,19 @@ exports.postCommentByReviewId = (req, res, next) => {
       next(err);
     });
 };
+
+exports.removeCommentById = (req, res, next) => {
+  const { comment_id } = req.params;
+  const promises = [
+    deleteCommentById(comment_id),
+    doesCommentIdExist(comment_id),
+  ];
+
+  return Promise.all(promises)
+    .then(() => {
+      res.status(204).send();
+    })
+    .catch((err) => {
+      next(err);
+    });
+};
```

Comment:
This functionality could be handled in a model with a Promise.reject for rows of 0 length, we want to try and make as few requests to the db as possible to ensure that our site works as quickly as possible if it is scaled up. For the purpose of this project it won't make a big difference in performance, but I'd consider updating this for performance.